tasks/provision-install-test/operator-pod-restart-task.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: operator-pod-restart
+spec:
+  description: Restart Kuadrant/RHCL operator pod to make sure everything gets reconciled successfully
+  params:
+    - name: kubeconfig-path
+      type: string
+  steps:
+    - computeResources:
+        limits:
+          cpu: '100m'
+          memory: 128Mi
+      env:
+        - name: KUBECONFIG
+          value: $(params.kubeconfig-path)
+      image: quay.io/kuadrant/testsuite-pipelines-tools:latest
+      imagePullPolicy: Always
+      name: operator-pod-restart
+      script: |
+        #!/usr/bin/env bash
+        set -evo pipefail
+
+        # Get the namespace of the Subscription (typically kuadrant-system but it can be different)
+        namespace=$(kubectl get subscription --all-namespaces -o json | jq -r '.items[] | select(.metadata.name == "kuadrant-operator") | .metadata.namespace')
+        if [[ -z "$namespace" ]]; then
+            echo "Subscription 'kuadrant-operator' not found in any namespace."
+            exit 1
+        fi
+
+        # Delete the operator Pod
+        pod_count=$(kubectl get pods -l app=kuadrant -o jsonpath='{.items[*].metadata.name}' -n $namespace | wc -w)
+        if [[ $pod_count -ne 1 ]]; then
+            echo "Expected exactly 1 pod, found $pod_count"
+            exit 1
+        fi
+        pod_name=$(kubectl get pods -l app=kuadrant -o jsonpath='{.items[0].metadata.name}' -n $namespace)
+        kubectl delete pod $pod_name --wait=true --timeout=60s -n $namespace
```

Comment:
Cleaner approach probably `kubectl rollout restart deployment kuadrant-operator-controller-manager -n kuadrant-system`

Reply:
I tried that one and it did not help. Not 100% what's going on there but apparently this is so clean (graceful) restart that the state is transferred in between the pods so the operator's reconcile is not started fresh (it remembers the fact that dns-/limitator-/authorino- operators were not installed and did not recover).

The hard restart (by deleting pod not this gracefully) is indeed needed to unblock the Kuadrant operator.